fetch.js

```diff
@@ -194,6 +226,9 @@
       }
 
       xhr.open(self.method, self.url)
+      if (has_responseType) {
```

Comment:
Happy to change this, but note the module scope feature detection would still be required for switching method implementations in `Body`.